docs/03-03-examples-alpha.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Kyma CLI alpha command usage examples
+type: Details
+---
+
+The following examples show how to provision a cluster, deploy Kyma, and <!--- update is not among the alpha commands ---> with alpha commands.
+
+## Provision a cluster locally or using cloud providers
+<!---provision gke is not among the alpha commands--->
+To provision a cluster on a specific cloud provider (in this example GCP), run:
+
+```bash
+kyma alpha provision gke --credentials {SERVICE_ACCOUNT_KEY_FILE_PATH} --name {CLUSTER_NAME} --project {GCP_PROJECT} 
+```
+
+To provision a Minikube cluster, run:
+<!---provision minikube is not among the alpha commands--->
+```bash
+kyma alpha provision minikube
+```
+
+## Deploy Kyma
+
+To deploy Kyma using your own domain, run:
+
+```bash
+kyma alpha deploy --domain {DOMAIN} --tls-cert {TLS_CERT} --tls-key {TLS_KEY}
+```
+
+`{TLS_CERT}` and `{TLS_KEY}` are the TLS certificate and TLS key for the domain used. Note that both of them must be encoded in base64. You can use [OpenSSL](https://www.openssl.org/) (which is installed by default with macOS) to convert your TLS certificate and TLS key to base64-encoded values.
+
+You can simplify the process to deploy Kyma with your own domainin  `{DOMAIN}` the following way:
+
+1. Encode the TLS certificate and TLS key, and export them as environment variables:
+  
+    ```bash
+    export TLS_CERT="$(openssl base64 -in {TLS_CERT_FILE_PATH})"
+    export TLS_KEY="$(openssl base64 -in {TLS_KEY_FILE_PATH})"
+    ```
+
+    `{TLS_CERT_FILE_PATH}` is the path to the file containing the TLS certificate, and `{TLS_KEY_FILE_PATH}` is the path to the file containing the TLS key.
+
+2. Use these environment variables in the command:
+
+    ```bash
+    kyma alpha deploy --domain {DOMAIN} --tls-cert $TLS_CERT --tls-key $TLS_KEY
+    ```
+
+To deploy Kyma from the `master` branch, run:
+
+```bash
+kyma alpha deploy --source master
+```
+
+To deploy Kyma using your own Kyma installer image, run:
+
+```bash
+kyma alpha deploy --source {IMAGE}
+```
+
+To build an image from your local sources and deploy Kyma on a remote cluster based on this image, run:
+
+```bash
+kyma alpha deploy --source local --custom-image {IMAGE}
```

Comment:
Picking as source the local checkout is still supported, but you cannot specify a custom image anymore as it is simply not required and possible anymore, we simplified a lot here

Reply:
When using the `local` source option, we should point out that the CLI will per default expect the sources in the `./workspace` folder  (which is probably not the location where the user stores the Kyma sources locally). He can adjust the workspace-path by setting the `-w ${path_to_workspace}` parameter.

Reply:
Alright! This is what I got now:

To build Kyma from your local sources and deploy it on a remote cluster, run:

```bash
kyma alpha deploy --source=local
```
> **NOTE:** By default, Kyma expects to find the sources in the `./workspace folder`. To adjust the path, set the `-w ${PATH_TO_WORKSPACE}` parameter.